cmake/modules/extensions.cmake

```
@@ -2572,7 +2617,7 @@ Please provide one of following: APPLICATION_ROOT, CONF_FILES")
   if(${ARGV0} STREQUAL APPLICATION_ROOT)
     set(single_args APPLICATION_ROOT)
   elseif(${ARGV0} STREQUAL CONF_FILES)
-    set(options REQUIRED)
+    set(options SOC REQUIRED)
```

`QUALIFIERS` is a better option name, as what you want to pick up is overlays / conf files based upon the qualifiers and not only the SoC (SoC being part of the qualifiers, so it's also covered by that name)

```suggestion
    set(options QUALIFIERS REQUIRED)
```

---

Updated